rim-page.html

```diff
@@ -0,0 +1,327 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Rim</title>
+        <style>
+            .container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: float;
+            }
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            .hero-image {
+                height: 100%;
+                background-image: url(fistbump-over-desk.jpg);
+                background-position: center;
+                background-repeat: no-repeat;
+                background-size: cover;
+                position: relative;
+            }
+            .hero-text {
+                text-align: left;
+                position: absolute;
+                top: 50%;
+                bottom: 50%;                
+            }           
+            .container-2 {            
+                width: 25%;
+            }
+            .container-3 {
+                position: relative;
+                text-align: left;
+                color: white;
+            }
+            .bb-footer {
+                float: left;
+                width: 20%;
+            }
+            .bb-footer li {
+                display: block;
+            }
+            .copyright {
+                display: block;
+                width: 100%;
+                float: right;
+            }    
+        </style>
+        <script>
+            console.log("Hello world!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <nav>
+                        <ul>
+                            <li><a href="#Logo">Logo</a></li>
+                            <li><a href="#Products">Products</a></li>
+                            <li><a href="#Solutions">Solutions</a></li>
+                            <li><a href="#Industries">Industries</a></li>
+                            <li><a href="#Partners">Partners</a></li>
+                            <li><a href="#Support">Support</a></li>
+                            <li><a href="#Blog">Blog</a></li>
+                            <li><a href="#Search">Search icon</a></li>
+                            <li><a href="#Headset">Headset icon</a></li>
+                        </ul>
+                    </nav>
+                </div>
+        </div>                                                    
+        <div class="container hero-image">
```

Comment:
Why does this have a class of container?  You already have this on line 58, which is the only div that should have container as a class.  Please review index.html (the file I sent you) carefully.

Reply:
This is a different section of the webpage so to target it give it its own container and class of “hero-image”.

Reply:
hero-image as a class is fine here, but container is not.